acquisitions/src/main/resources/thunderjet/mod-invoice/reusable/cancel-invoice-check-encumbrance-status.feature

```diff
@@ -0,0 +1,282 @@
+@ignored
+Feature: Cancel an invoice check encumbrance
+  # parameters: paymentStatusTable
+
+  Background: cancelInvoiceCheckEncumbrance
+    * url baseUrl
+
+  Scenario: Cancel an invoice check encumbrance
+    * def fundId = globalFundId
+    * def instanceId = call uuid
+    * def orderId = call uuid
+    * def poLineId = call uuid
+    * def paymentStatus = __arg.paymentStatus
+    * def encumbranceId = call uuid
+    * def invoiceId = call uuid
+    * def invoiceLineId = call uuid
+    * def isbn = "9780552142359"
+
+    * print "1. Create instances"
+    Given path 'inventory/instances'
+    And request
+      """
+      {
+        "id": "#(instanceId)",
+        "source": "FOLIO",
+        "title": "Interesting Times",
+        "instanceTypeId": "#(globalInstanceTypeId)",
+        "identifiers": [
+          {
+            "value": "#(isbn)",
+            "identifierTypeId": "#(globalISBNIdentifierTypeId)"
+          }
+        ]
+      }
+      """
+    When method POST
+    Then status 201
+
+    * print '2. Create composite order'
+    * def order =
+      """
+      {
+        "id": "#(orderId)",
+        "tags": {
+          "tagList": [
+            "amazon"
+          ]
+        },
+        "notes": [
+          "Check credit card statement to make sure payment shows up"
+        ],
+        "billTo": "5f8a321e-6b38-4d90-92d4-bf08f91a2242",
+        "shipTo": "f7c36792-05f7-4c8c-969d-103ac6763187",
+        "vendor": "#(globalVendorId)",
+        "approved": true,
+        "manualPo": false,
+        "poNumber": "10000",
+        "template": "4dee318b-f5b3-40dc-be93-cc89b8c45b6f",
+        "orderType": "One-Time",
+        "acqUnitIds": [],
+        "reEncumber": false,
+        "nextPolNumber": 2,
+        "workflowStatus": "Pending"
+      }
+      """
+    * remove order.poNumber
+    * remove order.billTo
+    * remove order.shipTo
+    Given path 'orders/composite-orders'
+    And request order
+    When method POST
+    Then status 201
+
+    * print '3. Create order lines'
+    * def poLine =
+      """
```

Comment:
we can use sample data and set required fill search for create poLine to see similar patterns
example:
```
    # 3. Verify success when creating an order line with correct fields and 'Physical Resource' order format
    * def poLine = read('classpath:samples/mod-orders/orderLines/minimal-order-line.json')
    * set poLine.id = poLineId1
    * set poLine.purchaseOrderId = orderId
    * set poLine.physical.createInventory = 'Instance, Holding, Item'
    * set poLine.isBinderActive = true
    * set poLine.orderFormat = 'Physical Resource'

```

Reply:
consider this refactoring for other places as well